cobc/pplex.l

```diff
@@ -1945,7 +2093,12 @@ start:
 	/* Skip comments that follow after AUTHORS, etc. */
 	if (unlikely (within_comment)) {
 		/* Check all of "Area A" */
-		for (ipchar = 7; ipchar < (n - 1) && ipchar < 11; ++ipchar) {
+		/* CHECKME: MF's doc on TERMINAL format reads "Area B starts in
```

Comment:
Did the check with ACU 3.2.1, result: It depends on the indicator width:

* if none specified (width = 0): starts at col 5
* if specified, for example a `-` (width = 1), starts at col 6
* for debugging lines with an indicator of `\D` (width = 2): starts at col 7

Reply:
The last commit - indicator width did actually fix that and only the testcase is missing, correct?
Do we have a chance to actually have that before "one of the next bigger PRs 'area a/b handling' as prerequisite for #35"?

Reply:
That last commit did handle indicator width the wrong way (width above 1 was counted as part of area A). I've thus added a test for the `TERMINAL` format to check this point, along with a fix.